source/ShopPanel.cpp

```diff
@@ -1413,6 +1466,10 @@ vector<ShopPanel::Zone>::const_iterator ShopPanel::Selected() const
 // letter of the button (or ' ' if it's not on a button).
 char ShopPanel::CheckButton(int x, int y)
 {
+	if(x > Screen::Right() - SIDEBAR_WIDTH - 342 && x < Screen::Right() - SIDEBAR_WIDTH - 316 &&
+		y > Screen::Bottom() - 31 && y < Screen::Bottom() - 4)
+		return 'f';
```

Comment:
We need to make this use interface data at some point. This is ugly. 🙈
You're fine to leave it as is now, just a problem for future us.

Reply:
Yeah I mentioned the uglyness of this above. The current code just isn't designed to have buttons on the left side of the screen 😬